src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/UserAccessService.kt

```diff
@@ -99,6 +99,12 @@ class UserAccessService(
   fun userCanManagePremisesOutOfServiceBed(user: UserEntity, premises: PremisesEntity) =
     userCanManagePremisesLostBeds(user, premises)
 
+  fun currentUserCanViewOutOfServiceBeds() =
+    userCanViewOutOfServiceBeds(userService.getUserForRequest())
+
+  fun userCanViewOutOfServiceBeds(user: UserEntity) =
+    user.hasRole(UserRole.CAS1_WORKFLOW_MANAGER)
```

Comment:
👍 this is correct (though is going to change to something clearer soon, e.g. `CRU_TEAM`)